pkgs/shells/zsh/zsh-vi-mode/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "zsh-vi-mode";
+  version = "0.8.5";
+
+  src = fetchFromGitHub {
+    owner = "jeffreytse";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "EOYqHh0rcgoi26eopm6FTl81ehak5kXMmzNcnJDH8/E=";
+  };
+
+  dontBuild = true;
+
+  installPhase = ''
+    mkdir -p $out
+    cp *.zsh $out/
```

Comment:
Looking at other zsh plugin derivations they all seem to put their files on `$out/share/<plugin-name>` instead, while currently your derivation is putting them on `$out/<plugin-name>`.

I think we should follow this pattern.

Reply:
I have seen that, and thought about it before submitting this MR. Being unfamiliar with ZSHs packaging and module system I was duplicating what the [install section says for `zsh-vi-mode`](https://github.com/jeffreytse/zsh-vi-mode#%EF%B8%8F-installation)

```bash
git clone https://github.com/jeffreytse/zsh-vi-mode.git $HOME/.zsh-vi-mode
source $HOME/.zsh-vi-mode/zsh-vi-mode.plugin.zsh
```
Thinking about it, there is no reason I shouldn't be able to package it under `$out/share/zsh-vi-mode` and follow he existing nix conventions.